t/140-ssl-c-api.t

```diff
@@ -811,3 +817,700 @@ lua ssl server name: "test.com"
 --- no_error_log
 [error]
 [alert]
+
+
+
+=== TEST 6: simple cert + private key with password
```

Comment:
@agentzh  Thanks for you suggestion, I will take a look at  `add_block_preprocessor`.

@ghedo what's your meaning? My plan is to add the new api at first. When we let it  merge to master  then creating another PR to `lua-resty-core`. Is that wrong?